frontend/src/components/AppSelect.vue

```diff
@@ -15,61 +14,67 @@
         adaptive-width
         strategy="fixed"
       >
-        >
-        <ListboxButton as="template">
+        <ListboxButton v-slot="{ open }" as="template">
           <AppButton
-            :icon="open ? faAngleUp : faAngleDown"
+            v-tooltip="tooltip"
+            :icon="open ? faCaretUp : faCaretDown"
             :flip="true"
-            class="button"
+            class="box"
+            :style="{ gridColumn: multi ? '' : 'span 2' }"
+            @keydown="onKeypress"
           >
-            {{ capitalize(selectedLabel) }}
+            {{ selectedLabel }}
+            <template #preview>
```

Comment:
Allow component to display preview slow of selected option, such as gradient or image. Also in each option in dropdown list, below.